test/custom-keystore.test.js

```diff
@@ -28,8 +28,9 @@ Object.keys(testAPIs).forEach(API => {
       rmrf.sync(dbPath)
       ipfsd = await startIpfs(API, config.daemon1)
       ipfs = ipfsd.api
-      orbitdb1 = new OrbitDB(ipfs, dbPath + '/1', {
-        keystore: CustomTestKeystore
+      orbitdb1 = await OrbitDB.createInstance(ipfs, {
+        directory: dbPath + '/1',
```

Comment:
We should prolly use path.join(dbPath, '1') here to make sure paths are joined correctly (also on Windows).